helm/sealed-secrets/Chart.yaml

```diff
@@ -1,11 +1,12 @@
 name: sealed-secrets
 description: A Helm chart for Sealed Secrets
-version: 1.13.2
+version: 1.14.0
 icon: https://avatars0.githubusercontent.com/u/34656521?s=200&v=4
 appVersion: 0.13.1
-kubeVersion: ">=1.9.0-0"
+kubeVersion: ">=1.16.0-0"
```

Comment:
currently we officially support 1.15, but I think we can drop support now; I have to update the CI environment and documentation too though first

Reply:
... yaks have to be shaven ... #492